README.md

```diff
@@ -61,7 +61,7 @@ parsePlugin.initialize(appId, clientKey, function() {
 Usage
 -----
 ```
-<script type="text/javascript">
+<script type="text/javascript>
```

Comment:
Yea... I think you just need to merge with master. I like what you did though!

ian

> On Apr 23, 2015, at 4:38 PM, ✌ <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -61,7 +61,7 @@ parsePlugin.initialize(appId, clientKey, function() {
> >  Usage
> >  -----
> >  ```
> > -<script type="text/javascript">
> > +<script type="text/javascript>
> I have no idea..definitely a mistake.
> 
> —
> Reply to this email directly or view it on GitHub.
>